cmd/handlers.c

```diff
@@ -178,6 +181,32 @@ cmdGetTime(CmdPacket *cmd) {
   return;
 }
 
+// Command handler for IP information
+static void ICACHE_FLASH_ATTR
+cmdGetWifiInfo(CmdPacket *cmd) {
+  CmdRequest req;
+
+  cmdRequest(&req, cmd);
+  if(cmd->argc != 0 || cmd->value == 0) {
+    cmdResponseStart(CMD_RESP_V, 0, 0);
+    cmdResponseEnd();
+    return;
+  }
+
+  uint32_t callback = req.cmd->value;
+
+  struct ip_info info;
+  wifi_get_ip_info(0, &info);
```

Comment:
This gets the STA info. Shouldn't this thing return both the STA and AP info? Or provide two calls for the two?